website/core/Head.js

```diff
@@ -19,6 +19,16 @@ const Head = React.createClass({
         <meta property="og:image" content="https://facebook.github.io/jest/img/opengraph.png" />
         <meta property="og:description" content={this.props.description} />
 
+
+        <link rel="apple-touch-icon" sizes="180x180" href="/jest/img/favicon/apple-touch-icon.png" />
+        <link rel="icon" type="image/png" sizes="32x32" href="/jest/img/favicon/favicon-32x32.png" />
+        <link rel="icon" type="image/png" sizes="16x16" href="/jest/img/favicon/favicon-16x16.png" />
+        <link rel="manifest" href="/jest/img/favicon/manifest.json" />
+        <link rel="mask-icon" href="/jest/img/favicon/safari-pinned-tab.svg" color="#5bbad5" />
```

Comment:
Is this how it supposed to look?

<img width="109" alt="screen shot 2017-05-12 at 12 01 14" src="https://cloud.githubusercontent.com/assets/5106466/25993614/d2809b28-370a-11e7-9132-52ad6d199f8c.png">